glib/gobject.go

```diff
@@ -236,7 +236,7 @@ func (v *Object) SetPropertyValue(name string, value *Value) error {
 		fallthrough
 	default:
 		if valType != propType {
-			return fmt.Errorf("Invalid type %s for property %s", value.TypeName(), name)
+			return fmt.Errorf("Invalid type %s for property %s(%s)", value.TypeName(), name, propType.Name())
```

Comment:
Specifying type name of property given into error message would be helpful for debugging I think.

```go
appSrc, err = gst.NewElement("appsrc")
err = appSrc.SetProperty("max-bytes", 1048576)
	if err != nil {
		logger.Errorf("gst appsrc err on set property max-bytes : %+v", err)
	}
```
property description (appsrc, max-bytes)
```
  max-bytes           : The maximum number of bytes to queue internally (0 = unlimited)
                        flags: readable, writable
                        Unsigned Integer64. Range: 0 - 18446744073709551615 Default: 200000
```
As is
`Invalid type gint for property max-bytes`

To be
`Invalid type gint for property max-bytes(guint64)`